docker/openproblems-python-extras/requirements.txt

```diff
@@ -8,8 +8,10 @@ umap-learn
 phate
 pyensembl
 pybedtools
+tangram-sc==1.0.2
 scvi-tools>=0.16.0
 torch
 git+https://github.com/michalk8/neuralee@master  # contains gradient error fix
 tensorflow-cpu==2.4.1
 git+https://github.com/czbiohub/molecular-cross-validation
+anndata>=0.7.6,<0.8
```

Comment:
This should be set in `setup.py` since it's a requirement for all of the images, not just the extras